images.md

```diff
@@ -0,0 +1,22 @@
+# Lambda Instance images creation
+
+## Description
+
+Using the fokia library, an administrator is able to create updated images for the Lambda Instance Master and Slave nodes.
+
+## Usage
+
+1. To create the Lambda Instance images, one must use the fokia library. This can be done either on your own computer, after installing the required dependencies, or using an up-to-date Lambda Service VM.
+2. Navigate to the fokia library folder: `cd <project_directory>/core/fokia`.
+3. Run `python lambda_instance_manager.py --image-creation --project-name <~okeanos project name> --auth_token <~okeanos auth token>`, to create a minimal Lambda Instance suitable for image creation.
+4. After lambda_instance_manager finishes the Lambda Instance creation successfully, a Lambda Master and a Lambda Slave node will be available on your cyclades account. You will be able to ssh  into their root user, from the system, where you run the lambda_instance_manager.
```

Comment:
Wait I am confused. Were the previous steps required in order to spin up the VMs for the Lambda Master and the Lambda slave, so that *afterwards* we can create the images?